### DISCUSSION TITLE
如何修改ping的触发周期 ？

### DISCUSSION THREAD

**Discussion by dev4u** [shadowsocks/shadowsocks-rust#638] [General]
`server_stat.rs`里面的`DEFAULT_CHECK_INTERVAL_SEC`，我能通过运行参数，或者配置文件传入具体的值吗？现在的周期是10s，这对于我来说，有点太频繁了。

**Comment by zonyitoo** [comment]
There is no way to customize the interval of ping. 10 seconds is already too long for most cases, I don't think it should be configurable.

**Reply by dev4u** [reply]
我用在手机环境下，10秒的周期是太频繁。在电源管理显示，耗电量真的不太理想，比微信的耗电还高。

**Reply by zonyitoo** [reply]
It will become an issue if you set the interval too long, for example, it cannot switch in time when a server down.

**Reply by zonyitoo** [reply]
I is already 10 seconds, why would that lead to high battery consumption. 10 seconds is already very long.

**Reply by dev4u** [reply]
我的诉求不全是灾备，更多的是将我的流量匀到数台小流量的节点上。
